internal/pkg/override/cdk.go

```diff
@@ -113,7 +113,7 @@ func (cdk *CDK) install() error {
 
 func (cdk *CDK) transform(body []byte) ([]byte, error) {
 	buildPath := filepath.Join(cdk.rootAbsPath, ".build")
-	if err := cdk.fs.MkdirAll(buildPath, fs.ModeDir); err != nil {
+	if err := cdk.fs.MkdirAll(buildPath, 0755); err != nil {
```

Comment:
What's the difference/which should we use when? (I see we have `0755` everywhere else in the codebase.)

Reply:
> tl;dr: just use `0755`.


when i had `fs.ModeDir` I got an access denied permission because this is the permission that it sets apparently:
```
ls -l                                                                                                                                 
d---------  2 karakuse  staff     64 Jan 19 12:55 test
```
So it looks like it marks it with no other permissions 😅 so I got an access denied when trying to write a file.

I then followed the permissions here for 0755 https://www.multacom.com/faq/password_protection/file_permissions.htm

The cool thing is that when using `fs.Mkdirall(path, 0755)` it automatically under the hood does: `fs.ModeDir | 0755` so you don't have to remember marking the file as a directory.
```
drwxr-xr-x  2 karakuse  staff    64B Jan 19 13:14 test
```